dtonator/src/main/java/com/bizo/dtonator/properties/Prop.java

```diff
@@ -6,21 +6,43 @@
   public final String name;
   public final String type;
   public final boolean readOnly;
-  public final String getterMethodName;
-  public final String setterNameMethod;
+  private final String getterMethodName;
```

Comment:
Given this is just an internal DTO and not a public API, I generally prefer the sacrilege of just using public final fields. Especially since some of the fields work that way already, it'd be better to be consistent that have some public fields and some private fields+getters.

Reply:
You know the way these fields are meant to be used, and now I have a pretty good idea, but someone else who forks this code later might not. Making the fields private and read-only makes it crystal clear that nothing should change those properties once they are set. 

The best way to do it would actually be to store the Method itself and make the getters access the properties of the method. Unfortunately, that would make testing really hairy, so I don't think that is an option.